src/hardware/vga_draw.cpp

```diff
@@ -73,37 +73,123 @@ static Bit8u * VGA_Draw_2BPPHiRes_Line(Bitu vidstart, Bitu line) {
 	return TempLine;
 }
 
-static Bitu temp[643]={0};
+static Bit8u byte_clamp(int v) {
+	v >>= 13;
+	return v < 0 ? 0 : (v > 255 ? 255 : v);
+}
 
```

Comment:
future note: C++17 has `std::clamp()`

Reply:
This is a very unique clamp that divides the inbound value by 8192 and then clamps the result.

Reply:
yep, was thinking `return v < 0 ? 0 : (v > 255 ? 255 : v);` could become eg. `return std::clamp(v,0,255);`

Reply:
(d'oh.. indeed!)